docs/en/usage/patterns/multiwin.md

```diff
@@ -5,10 +5,6 @@ title: Multiwin
 import Alert from '@theme/Alert'
 import useBaseUrl from '@docusaurus/useBaseUrl'
 
-<Alert type="warning" icon="info-alt" title="Please note">
```

Comment:
@nothingismagick can you update the diagram? maybe just replace glutin with webview for now